sys/include/sched_round_robin.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2020 TUBA Freiberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sche_round_robin Round Robin Scheduler
+ * @ingroup     sys
+ * @brief       This module module provides round robin scheduling within
+ *              for all runnable threads within each not masked priority.
+ *              Priority 0 is masked by default.
+ *              There is no fainess involved in this scheuler. It schedules,
+ *              when the timer ticks. Keep it as simple as possible.
```

Comment:
Isn't "scheduling when the timer ticks" as fair as it can get?

Reply:
kind of yes/no as fair as it can get with such simple methods, but not fair as in: getting a fair share of the cpu time.
Cause there are some CPU usage patterns that may screw that up.